composer.json

```diff
@@ -16,24 +16,21 @@
 		"WordPress standards",
 		"DekodeInteraktiv"
 	],
-	"require": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-		"phpcompatibility/phpcompatibility-wp": "~2.1.3",
-		"wp-coding-standards/wpcs": "dev-develop"
-	},
 	"require-dev": {
-		"ergebnis/composer-normalize": "^2.29"
+		"ergebnis/composer-normalize": "~2.28.3",
```

Comment:
Moving the dependencies to `dev` causes them to be missing in the project that requires this package.

Reply:
Moved back to `require`. Thank you for the info.